{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/local.py

```diff
@@ -23,7 +23,7 @@
     'default': {
```

Comment:
So now that we have this environ stuff going which is great! But that means that basically all of our environment variable settings can be moved to `base.py`. And the `local.py` or whatever we are calling it can then just have stuff like backends and template options.

In particular, `SECRET_KEY` and `STRIPE_*_KEY` should be moved to `base.py` and `STRIPE_*` should be behind the cookiecutter flag.